azkaban-exec-server/src/main/java/azkaban/execapp/FlowRunner.java

```diff
@@ -1464,6 +1458,15 @@ public FlowRunnerEventListener() {
       metaData.put("startTime", String.valueOf(flow.getStartTime()));
       metaData.put("submitTime", String.valueOf(flow.getSubmitTime()));
 
+      // project upload info
+      final ProjectFileHandler handler = flowRunner.projectFileHandler;
+      if (handler != null) {
+        metaData.put("uploadUser", handler.getUploader());
+        metaData.put("uploaderIpAddr", handler.getUploaderIpAddr());
```

Comment:
Should we add prefix "_projectFile_" to "uploadUser" and "uploaderIpAddr" properties to maintain consistency with the rest of them?

Reply:
It was becoming too big, but I dig it.